golem-worker-executor-base/Cargo.toml

```diff
@@ -43,7 +43,8 @@ fs-set-times = "0.20.1"
 futures = { workspace = true }
 futures-util = { workspace = true }
 gethostname = "0.4.3"
-golem-wit = { version = "0.2.3" }
+#golem-wit = { version = "0.2.3" }
+golem-wit = { git = "https://github.com/golemcloud/golem-wit.git", rev = "5079afa1" }
```

Comment:
Just adding a review comment here to make sure it's not fogotten :)

Reply:
updated